### ISSUE TITLE
Nvidia APP Just loads

### ISSUE THREAD

**Issue by ignhakka** [NVIDIA/trt-llm-rag-windows#99]
I have reinstalled the app 5 times, force closed it restarted my pc. Idk what else to do. I have a 4070 ti super and i just cant use the app![Image](https://github.com/user-attachments/assets/845077a2-d191-452a-8b2f-f54d0bc7a21d)

**Comment by doctorsangria** [comment]
#90

**Comment by ZombieWorm** [comment]
I bought a 5080 and installed it this week. I went to the nvidia App and it loads the tool. However if I try to select another local folder it doesn't take. When I ask the Chat a question I get "Problem generating response: Data source may be empty or unsupported – Ensure dataset compatibility with the AI model".

**Comment by anujj** [comment]
0.4v version is not supported on 5080 and 5090

**Comment by akaanupkumar** [comment]
The 0.5 version would work on RTX 5080. Please try it out.

**Comment by ZombieWorm** [comment]
Thanks Akaan, will give this a go.

**Comment by ZombieWorm** [comment]
The installer failed on the model llama. Is there any way to continue the installer do I just run the app exe again?

**Comment by akaanupkumar** [comment]
Can you tell what the failure message was ? Its better to clean up and restart exe.

User guide available in below link mention troubleshooting steps for failures:

https://www.nvidia.com/en-in/ai-on-rtx/chatrtx/

**Comment by ZombieWorm** [comment]
It installed the workbench, it failed on the llama NIM step.

**Comment by ZombieWorm** [comment]
OK I found the issue. I left it installing unattended so could not see the output till this morning. It must have ran out of disk space as it was getting tight.

I cleared loads of headroom space this time round and it all completed successfully.

**Comment by ZombieWorm** [comment]
Is there a direct way to address this message when you use the Data folder as a source to chat about the response is: 

"Problem generating response: Data source may be empty or unsupported – Ensure dataset compatibility with the AI model"

I know in old version I had to manually update a python file regarding pooling. Can you help?

**Comment by akaanupkumar** [comment]
Glad to know the original installation issue is resolved. Regarding this data source error:

1. can you confirm that supported datatype files are actually present in folder selected ? 
2. Can you run the ChatRTX exe from the cmd to see the logs. It would give more logs related to what is happening. ( Default location for the ChatRTX exe (NVIDIA ChatRTX.exe) is : C:\Program Files\NVIDIA Corporation\ChatRTX\RAG\trt-llm-rag-windows-ChatRTX_0.5.0\ChatRTXUI\dist)

**Comment by ZombieWorm** [comment]
1. confirm I tried other folders too with content inside them.

2. I start the command line but the app has spinning central logo with Loading ChatRTX...

Where can I view the logs?

**Comment by akaanupkumar** [comment]
The logs will be available in the cmd you started the ChatRTX exe. If that reports some error you might see this behavior.

**Comment by ZombieWorm** [comment]
Hi sir, I do this and the chat gui window doesnt progress "Loading ChatRTX" there are no logs in the command line (ran in administrator).

**Comment by akaanupkumar** [comment]
That is surprising. Is there nothing printed on cmd at all?

If you highlight/select text on cmd, the execution is stopped. Can you verify execution is not halted because of that?
( https://www.reddit.com/r/sysadmin/comments/118l8s0/am_i_crazy_or_does_terminal_progress_sometimes/)

**Comment by ZombieWorm** [comment]
I was running the .exe from the folder a level before (dist). That is a different exe.

I found the correct exe you wanted me to run in the "win-unpacked" folder.

OK so here is the log after I ask it to check my dataset to return an answer (still getting the problem generating response):

23:10:25.256 > getDatasetInfo call returned  {"sources": ["directory", "nodataset"], "selected": "directory", "path": "C:\\ProgramData\\NVIDIA Corporation\\ChatRTX\\sample_data\\dataset", "path_chinese": "C:\\ProgramData\\NVIDIA Corporation\\ChatRTX\\sample_data\\chinese_dataset", "path_clip": "C:\\ProgramData\\NVIDIA Corporation\\ChatRTX\\sample_data\\images_dataset", "selected_path": "C:\\tmp\\receipts"}
23:10:25.257 > getSampelQuestionInfo returned  {"default": {"modelIdList": ["mistral_7b_AWQ_int4_chat", "llama2_13b_AWQ_INT4_chat", "gemma_7b_int4", "meta/llama-3.1-8b-instruct", "mistral-nemo-12b-instruct"], "dataset_path": "C:\\ProgramData\\NVIDIA Corporation\\ChatRTX\\sample_data\\dataset", "isRelative": true, "queries": ["How does NVIDIA ACE generate emotional responses?", "What is Portal prelude RTX?"]}, "chinese": {"modelIdList": ["chatglm3_6b_AWQ_int4"], "dataset_path": "C:\\ProgramData\\NVIDIA Corporation\\ChatRTX\\sample_data\\chinese_dataset", "isRelative": true, "queries": ["NVIDIA ACE\u662f\u5982\u4f55\u751f\u6210\u5bcc\u6709\u60c5\u611f\u7684\u56de\u590d\u7684\uff1f", "\u4f20\u9001\u95e8\uff1a\u5e8f\u66f2\u662f\u4ec0\u4e48\uff1f", "\u534a\u6761\u547d 2 RTX\u7248\u6709\u4ec0\u4e48\u91cd\u8981\u610f\u4e49\uff1f", "\u745e\u5947\u4e0e\u53ee\u5f53\uff1a\u65f6\u7a7a\u8df3\u8f6c\u4f55\u65f6\u4f1a\u5728PC\u5e73\u53f0\u53d1\u5e03\uff1f"]}, "images": {"modelIdList": ["clip_model"], "dataset_path": "C:\\ProgramData\\NVIDIA Corporation\\ChatRTX\\sample_data\\images_dataset", "isRelative": true, "queries": ["Pictures of bicycles", "Pictures of toys", "Pictures of dinosaurs", "Pictures of computer"]}}
23:10:25.257 > call initChatbotEngine returned response  null
Copied contents of C:\Program Files\NVIDIA Corporation\ChatRTX\env_nvd_rag\lib\site-packages\ChatRTX\model_manager\../sample_data to C:\ProgramData\NVIDIA Corporation\chatrtx\sample_data
Copied contents of C:\Program Files\NVIDIA Corporation\ChatRTX\env_nvd_rag\lib\site-packages\ChatRTX\NIM_licenses to C:\ProgramData\NVIDIA Corporation\chatrtx\NIM_licenses
Batches:   0%|          | 0/1 [00:00<?, ?it/s]
Batches: 100%|ÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔûê| 1/1 [00:01<00:00,  1.90s/it]
Batches: 100%|ÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔûê| 1/1 [00:01<00:00,  1.90s/it]

**Comment by akaanupkumar** [comment]
Yes you are now running the correct exe and these were the logs I was requesting. But above looks like is only part of logs which does not mention the error. Could you look through full logs if you see any logs mentioning 'ERROR' or similar.
